src/WebApp/Startup.cs

```diff
@@ -69,32 +69,6 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapRazorPages();
             });
-
-            SeedDatabase(app);
-        }
-
-        private static void SeedDatabase(IApplicationBuilder app)
-        {
-            using (var scope = app.ApplicationServices.CreateScope())
-            {
-                var users = scope.ServiceProvider.GetService<UserManager<ApplicationUser>>();
-                var db = scope.ServiceProvider.GetService<ApplicationDbContext>();
-
-                // Todo. Use database migrations in the future.
-                db.Database.EnsureCreated();
```

Comment:
This line has to stay.

Reply:
> we do database first don't we
I wish I knew. 
I'd rather do db first to be honest so I guess lets do that.
I can just add the seed for the migration

Reply:
> I can just add the seed for the migration.

I did not know about that feature, it would have been so much better for the `admin@codidact` thing. Regardless, that is not necessary, the only line that needs to stay is:

~~~csharp
db.Database.EnsureCreated();
~~~

If we use database migrations we would have to change this to:

~~~csharp
db.Database.Migrate();
~~~

Reply:
https://github.com/codidact/authentication/pull/9/commits/bac0cc7e6e86918c747c56db8f97c97656127281